App/src/views/RequestsView.js

```diff
@@ -1,7 +1,49 @@
+/** @jsx jsx */
 import React from "react";
+import { jsx } from "@emotion/core";
+import Nabvar from "../components/Nabvar";
+import { Redirect } from "@reach/router";
+import { useUser } from "../contexts/user";
 
+const REQUESTS = [
```

Comment:
The variable 'REQUEST', could be in lower case

Reply:
Don't forget to implement add these properties about the model :  t.date "creationDate", "requester_id","requested_id", "rol", "status", "date_Shift","current_Shift_id", "requested_Shift_id"